lib/ansible/modules/remote_management/manageiq/manageiq_key.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Alex Braverman <abraverm@redhat.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.0',
+    'supported_by': 'community',
+    'status': ['preview']
+}
+
+DOCUMENTATION = '''
+---
+module: manageiq_key
+version_added: "2.4"
+description:
+    - create and import encryption key using the appliance_console_cli
+short_description: create and import encryption key
+author: "Alex Braverman Masis (@abraverm)"
+requirements:
+    - appliance_console_cli
+options:
+    fetch_key:
+        description: to fetch or not encryption key
```

Comment:
All `descriptions` should be full sentences, eg with capital letters and full stops.